types/liquid-pid/liquid-pid-tests.ts

```diff
@@ -0,0 +1,21 @@
+import * as PIDController from "liquid-pid";
```

Comment:
Your test should use (at a minimum) the code from the project's README. It says
```ts
var
  liquidPID = require('liquid-pid'),
  actualP = 0,
  pidController;

pidController = new liquidPID({
  temp: {
```

which will fail given the way you've written this .d.ts file